api/PclusterApiHandler.py

```diff
@@ -171,6 +172,7 @@ def get_app_config():
   return {
     "auth_url": AUTH_URL,
     "client_id": CLIENT_ID,
+    "oidc_provider": OIDC_PROVIDER,
```

Comment:
suggestion(non-blocking): Since we know which OIDC providers we are supporting, couldn't we use [python enums](https://docs.python.org/3/library/enum.html) instead of relying on strings?

Plus, the enum class [already provides an easy way](https://stackoverflow.com/a/41409392/3973689) of obtaining an enum from a string

Reply:
I wouldn't make this change at this point: at the moment, we get this value from the environment variable and the backend doesn't have a list of supported providers, this is the `AllowedValues` of the `OIDCProvider` parameter of the `pcluster-manager.yaml` Cfn template.